frontend/src/pages/Projects/Project/CloseOut/index.tsx

```diff
@@ -0,0 +1,174 @@
+    return project.data.data;
+  };
+
+  // Queries
+  // todo: Define a good type. "Any" type temporarily permitted.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const projectQuery: any = useQuery(`project - ${projectId}`, getProject, {
+    refetchOnWindowFocus: false,
+    retryOnMount: false,
+    refetchOnReconnect: false,
+    retry: false,
+    staleTime: Infinity,
+  });
+
+  const readFields = [
+    {
+      width: "full",
+      title: "Close out date",
+      value: projectQuery?.data?.close_out_date,
+    },
+    {
+      width: "full",
+      title: "Completed by",
+      value: projectQuery?.data?.completed_by_contact_id,
+    },
+    {
+      width: "full",
+      title: "Actual completion date of project",
+      value: projectQuery?.data?.actual_completion_date,
+    },
+    {
+      width: "full",
+      title: "Post implementation hand-off to operation completed",
+      value: projectQuery?.data?.hand_off_to_operations,
+    },
+    {
+      width: "full",
+      title: "Project documentation filled in accordance with records management",
+      value: projectQuery?.data?.records_filed,
+    },
+    {
+      width: "full",
+      title: "Contract evaluation completed if applicable",
+      value: projectQuery?.data?.contract_ev_completed,
+    },
+    {
+      width: "full",
+      title: "Contractor IDIR terminated / building passes returned",
+      value: projectQuery?.data?.contractor_security_terminated,
+    },
+  ];
+
+  const editFields: IEditFields[] = [
+    {
+      fieldName: "close_out_date",
+      fieldLabel: "Close out date",
+      fieldType: "date",
+      width: "full",
+    },
+    {
+      fieldName: "completed_by_contact_id",
+      fieldLabel: "Completed by",
+      fieldType: "singleText",
+      width: "full",
+      // tableName: "contacts",
+    },
+    {
+      fieldName: "actual_completion_date",
+      fieldLabel: "Actual completion date of project",
+      fieldType: "date",
+      width: "full",
+    },
+    {
+      fieldName: "hand_off_to_operations",
+      fieldLabel: "Post implementation hand-off to operation completed",
+      fieldType: "select",
+      width: "full",
+      tableName: "generic",
+    },
+    {
+      fieldName: "records_filed",
+      fieldLabel: "Project documentation filled in accordance with records management",
+      fieldType: "select",
+      width: "full",
+      tableName: "generic",
+    },
+    {
+      fieldName: "contract_ev_completed",
+      fieldLabel: "Contract evaluation completed if applicable",
+      fieldType: "select",
+      width: "full",
+      tableName: "generic",
+    },
+    {
+      fieldName: "contractor_security_terminated",
+      fieldLabel: "Contractor IDIR terminated / building passes returned",
+      fieldType: "select",
+      width: "full",
+      tableName: "generic",
+    },
+  ];
+
+  let content = <></>;
+  switch (projectQuery.isLoading) {
```

Comment:
isn't this the purpose of the Renderer component?

Reply:
That's true, I'll use Renderer here.